.github/workflows/test.yml

```diff
@@ -28,6 +28,11 @@ jobs:
           cache: true
           cache-dependency-path: go.sum
 
+      - name: Go Module tidy check
```

Comment:
This probably should not be here. You might need update your fork with the latest changes in origin, then rebase your code.

Reply:
Indeed, thank you for noticing - fixed.